test/boulder-tools/build.sh

```diff
@@ -25,7 +25,7 @@ apt-get install -y --no-install-recommends \
   automake
 
 PROTO_ARCH=x86_64
-if [ "${GOARCH}" = arm64 ]
+if [ "${1}" = linux/arm64 ]
```

Comment:
TARGETPLATFORM is an ENV so we can access it directly rather than passing as an arg.

Reply:
Oh, whoops, thanks!